minecraft_servers/northsoft/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+    "server_name": "northsoft",
+    "nice_name": "NorthSoft Netzwerk",
+    "direct_ip": "northsoft.eu",
+    "supported_languages": [
+       "de"
+    ],
+    "social": {
+      "discord": "https://dsc.gg/northsoft",
```

Comment:
Custom links are reserved for partners only; use a normal discord.gg/[YOUR-INVITE] link